src/index.js

```diff
@@ -1,17 +1,17 @@
-import React from 'react';
-import ReactDOM from 'react-dom/client';
-import './index.css';
+import React from "react";
+import ReactDOM from "react-dom";
+import { Provider } from "react-redux";
+import { store } from "./store/store";
 import App from './App';
-import reportWebVitals from './reportWebVitals';
+import Container from './components/loginform/modules/Container';
+import LoginForm from './components/loginform/LoginForm';
 
-const root = ReactDOM.createRoot(document.getElementById('root'));
-root.render(
-  <React.StrictMode>
+
+ReactDOM.render(
+  <Provider store={store}>
```

Comment:
If you resolve conversation that mean you fix comment. In this case I don't see any fix. And also I need to mention this is not first time. Please change it because it is not a good habit. Sometimes you will get away without me noticing that you did it, but the goal of your internship is to teach you to work and not to learn to hide from me